### PR TITLE
feat: replace wfh for car loan dataset on docs

### PR DESCRIPTION
**Pull request by alexnanny**
- Update documentation to replace the old `synthetic_binary_classification_dataset` (work from home dataset) in favor of the `synthetic_car_loan_dataset` 

- Update documentation on 12 pages:
  - Quick Start
    - https://7476f2e0.alex-nanny-branches.pages.dev/quick
  - Tutorial >> Data requirements
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/data_requirements#columns
  - Tutorial >> Estimating Performance >> Binary Classification
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/performance_estimation/binary_performance_estimation
  - Tutorial >> Realized Performance >> Binary Classification
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/performance_calculation/binary_performance_calculation
  - Tutorial >> Univariate Drift Detection
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/detecting_data_drift/univariate_drift_detection
  - Tutorial >> Multivariate Drift Detection
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/detecting_data_drift/multivariate_drift_detection
  - Tutorial >> Ranking
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/ranking
  - Tutorial >> Storing and Loading Calculators
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/storing_and_loading_calculators
  - Tutorial >> Working with results
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/working_with_results
  - Tutorial >> Adjusting Plots
    - https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/adjusting_plots
  - How it works >> Ranking >> Correlation Ranking
    - https://7476f2e0.alex-nanny-branches.pages.dev/how_it_works/ranking#correlation-ranking
  - CLI >> Configuration File
    - https://7476f2e0.alex-nanny-branches.pages.dev/cli/configuration_file#column-mapping-section

### CODE REVIEW

File: docs/tutorials/data_requirements.rst [hunk 1]
```diff
@@ -150,7 +123,7 @@ Target
 
 The actual outcome of the event the machine learning model is trying to predict.
```
Comment (nnansters) [inline docs/tutorials/data_requirements.rst:124]: `/_static/tutorials/data-requirements-index-based-x-axis.svg` and `/_static/tutorials/data-requirements-time-based-x-axis.svg` are need to be refreshed.

File: docs/tutorials/performance_calculation/binary_performance_calculation.rst [hunk 2]
```diff
@@ -46,10 +49,10 @@ Next a :class:`~nannyml.performance_calculation.calculator.PerformanceCalculator
...
-    * ``f1`` 
-    * ``precision`` 
+    * ``f1``
+    * ``precision``
     * ``recall``
-    * ``specificity`` 
+    * ``specificity``
     * ``accuracy``
     * ``confusion_matrix``
         * Normalization options: ``None`` (returns counts), ``true`` (normalize by true class of observations), ``pred`` (normalize by predicted class of observations), ``all`` (normalize by all observations).
```
Comment (nnansters) [inline docs/tutorials/performance_calculation/binary_performance_calculation.rst:58]: Alignment looks weird, probably better nesting under the list right above

File: docs/tutorials/performance_estimation/binary_performance_estimation.rst [hunk 3]
```diff
@@ -66,7 +70,7 @@ can check the :ref:`setting up page<chunking>` and :ref:`advanced guide<chunk-da
```
Comment (nnansters) [inline docs/tutorials/performance_estimation/binary_performance_estimation.rst:70]: Alignment looks weird, probably better nesting under the list right above

File: docs/tutorials/detecting_data_drift/univariate_drift_detection.rst [hunk 4]
```diff
@@ -9,7 +9,7 @@ Just The Code
 -------------
 
 .. nbimport::
-    :path: ./example_notebooks/Tutorial - Drift - Univariate.ipynb
+    :path: ./example_notebooks/Tutorial - Drift - Univariate - Car Loan.ipynb
```
Comment (nnansters) [inline docs/tutorials/detecting_data_drift/univariate_drift_detection.rst:12]: I just noticed we still have `integer` values in the `y_pred` column. That forces us to explicitly set that column as categorical before running NannyML. This was one of the things we didn't want to have any longer in the tutorials. I must've missed that one, sorry.

This does impact some of the notebooks, so I'll make the change to the dataset and the relevant notebooks tomorrow (remove the explicit conversion to `category` datatype)

File: docs/how_it_works/ranking.rst [hunk 5]
```diff
@@ -60,9 +60,9 @@ as shown in this plot below.
 
 .. nbimport::
```
Comment (nnansters) [inline docs/how_it_works/ranking.rst:61]: This code snippet contains an error (noticed it on main and fixed there, but we might as well apply it here:


```
# ranker fits on one metric and reference period data only

ranker2.fit(

    realized_perf_results.filter(period='reference', metrics=['recall']))

# ranker ranks on one drift method and one performance metric

correlation_ranked_features2 = ranker2.rank(

    univariate_results.filter(period='analysis', methods=['jensen_shannon']),

    realized_perf_results.filter(period='analysis', metrics=['recall']),

    only_drifting = False)
```

**Comment by nnansters** [comment]
@nikml awaiting your review

**Comment by nikml** [comment]
Adjusting plots
https://7476f2e0.alex-nanny-branches.pages.dev/tutorials/adjusting_plots

needs to have the x0 and x1 dates adjusted for the date indicator in order to match the behavior shown currently

https://nannyml.readthedocs.io/en/latest/tutorials/adjusting_plots.html

@alexnanny

**Comment by nikml** [comment]
Actually fix is simple enough, I 'll commit it when I save my changes.

**Comment by nikml** [comment]
PR is ready to be merged. Remaining doc issues are not cause of this PR.

**Pull request merged**